tests/test_flow_graph.py

```diff
@@ -1,56 +1,52 @@
-    assert 'if_task -> switch_task' in graph
-    assert 'switch_task -> end' in graph
-
-
-def test_create_graph_svg():
-    call_command('flow_graph', "{}".format(FLOW_PATH), svg=True)
-    source_file_name = "{}".format(FLOW_PATH)
-    svg_file_name = "{}.svg".format(FLOW_PATH)
-    source_file_path = os.path.join(dirname(dirname(abspath(__file__))), source_file_name)
-    svg_file_path = os.path.join(dirname(dirname(abspath(__file__))), svg_file_name)
-    try:
-        assert isfile(source_file_path)
-        assert isfile(svg_file_path)
-        with open(source_file_path, 'r') as f:
-            f.seek(0)
-            graph = f.read()
-            assert 'start -> savable_task' in graph
-            assert 'savable_task -> if_task' in graph
-            assert 'if_task -> switch_task' in graph
-            assert 'switch_task -> end' in graph
-    finally:
-        os.remove(source_file_path)
-        os.remove(svg_file_path)
-
-
-def test_create_graph_wrong_name():
-    """Test exceptions"""
-    with pytest.raises(CommandError):
-        call_command('flow_graph', "wrong_path.TestFlow")
-
-    with pytest.raises(CommandError):
-        call_command('flow_graph', "wrong_path")
-
-    with pytest.raises(CommandError):
-        call_command('flow_graph', "tests.testapp.flows.WrongFlow")
-
-    with pytest.raises(CommandError):
-        call_command('flow_graph', "wrong_path")
+
+class TestFlowGraph:
+    flow_path = 'tests.testapp.flows.TestFlow'
+
+    def test_create_default(self):
+        with StringIO() as stdout:
+            call_command('flow_graph', self.flow_path, stdout=stdout)
+            stdout.seek(0)
+            graph = stdout.read()
+        assert '<svg xmlns="http://www.w3.org/2000/svg" ' in graph
+
+    def test_svg(self):
+        with StringIO() as stdout:
+            call_command('flow_graph', '--type=svg', self.flow_path, stdout=stdout)
+            stdout.seek(0)
+            graph = stdout.read()
+        assert '<svg xmlns="http://www.w3.org/2000/svg" ' in graph
+
+    def test_bpmn(self):
+        with StringIO() as stdout:
+            call_command('flow_graph', '--type=svg', self.flow_path, stdout=stdout)
+            stdout.seek(0)
+            graph = stdout.read()
+        assert '<svg xmlns="http://www.w3.org/2000/svg" ' in graph
+
+    def test_output(self):
+        file_name = os.path.join(tempfile.mkdtemp(), 'test.svg')
+        with StringIO() as stdout:
+            call_command('flow_graph', '--output=%s' % file_name, self.flow_path, stdout=stdout)
+            stdout.seek(0)
+            graph = stdout.read()
+        assert graph == ''
+        assert os.path.exists(file_name)
+        with open(file_name) as fs:
+            graph = fs.read()
+        assert '<svg xmlns="http://www.w3.org/2000/svg" ' in graph
+
+    def test_create_graph_wrong_name(self):
+        """Test exceptions"""
+        with pytest.raises(CommandError):
+            call_command('flow_graph', "wrong_path.TestFlow")
```

Comment:
Here the 3 different wrong paths ("wrong_path.TestFlow", "wrong_path" and "tests.testapp.flows.WrongFlow", because "wrong_path" was actually duplicated) are different error conditions and are handled by raising exceptions in different places. So removing them decreases coverage.